python/lsst/ts/hvac/hvac_csc.py

```diff
@@ -249,12 +269,32 @@ async def handle_summary_state(self):
     async def configure(self, config):
         self.config = config
 
+    def _get_topic_enabled_state(self, topic):
```

Comment:
Please document what this does and what a `topic` is.

The term `topic` seems overloaded since it usually refers to SAL topics (or the salobj interface to them -- instances of salobj.topic.<Something>Topic named cmd_foo, evt_bar, etc.).

Reply:
Good point. Done.